README.rst

```diff
@@ -22,7 +22,7 @@ There are a few things to install in order to use this package:
   likely need to request it from your system admin. 
   Refer to https://docs.sylabs.io/guides/4.1/admin-guide/installation.html
 
-.. highlight:: console
+.. code-block:: console
```

Comment:
Is this for rendering the following text as code rather than bold or whatever highlight is?

Reply:
Actually, `.. highlight:: console` _also_ make the lines look like code and rendered correctly on github. But for some reason sphinx isn't able to autodoc it for readthedocs but seems to not have an issue with `.. code-block:: console`. They seem to render nearly identically (aside from some highlighting differences) 🤷‍♂️ 
![image](https://github.com/user-attachments/assets/3a7a752e-be1a-4511-98ff-6086343510b8)